src/patternfly/components/Button/docs/code.md

```diff
@@ -4,21 +4,22 @@ Always add a modifier class to add color to the button.
 
 ## Accessibility
 
-An active state button will appear pressed when active. There’s no need to add a class to `<button>`s as they use a pseudo-class. However, if you are using a button as a toggle, you can still force the same active appearance with `pf-is-active` (and include the `aria-pressed="true"`) should you need to replicate the state programmatically.
-
-Disabled buttons should include the `disabled="disabled"` attribute to indicate the state of the element to assistive technologies.
+| Attribute | Applied | Outcome |
+| -- | -- | -- |
+| `disabled="disabled"` | `.pf-c-button` | Indicates the disabled state of the element to assistive technologies |
+| `aria-pressed="true or false"` | `.pf-c-button` | If you are using a button as a toggle, you can still force the same active appearance with `pf-is-active` (and include the `aria-pressed=""`) should you need to replicate the state programmatically. |
```

Comment:
In the context of this table, I'd reword **Outcome** to something like this:

> Indicates that the button is a toggle. When set to "true", `pf-is-active` should also be set so that the button displays in an active state.